switch/switch_main.ml

```diff
@@ -176,9 +176,17 @@ let make_server config =
       >>= fun qs ->
       on_disk_queues := qs;
       info "Reading the redo-log from %s" redo_log_path;
-      let open Lwt_result in
-      Block.connect ("buffered:" ^ redo_log_path)
+      Block.connect ~buffered:true redo_log_path
+      >>= fun block -> begin
+        match block with
+        | `Ok block -> Lwt.return block
+        | `Error e ->
```

Comment:
This sounds like something that would be useful in every error and debugging message we emit.